### DISCUSSION TITLE
how to set storageState in this framework

### DISCUSSION THREAD

**Discussion by rajmicro78** [akshayp7/playwright-typescript-playwright-test#53] [Q&A]
I check the framework and not sure where i can set the storageState. Basically i want to skip login everytime.

**Comment by akshayp7** [comment]
Thank you for your patience. Have you tried this
https://www.youtube.com/watch?v=WHyQsX0w_5g

**Comment by sqagithub** [comment]
**storageState**
Can you give some example to reference on how to use   **test.use({ storageState: `auth/user.json` });**
in Page Object model? 


I have created in a separate process where my script stores the storage state. This part works. Now using test.use({}) above does not work.    
**const authFile = `auth/user.json`;**
 **await this.context.storageState({ path: authFile });**

**Comment by akshayp7** [comment]
Thanks for your patience can you try the below code 

const { saveStorageState } = require('./auth');

test.describe('Authenticated User Tests', () => {

    test.use({
        storageState: 'auth/user.json' // This path points to your saved storage state
    });

    test('should access the dashboard after login', async ({ page }) => {
     
        await page.goto('https://google.com');

    });
